src/GNDStk/test/GNDStk.test.cpp

```diff
@@ -2,7 +2,3 @@
 #define CATCH_CONFIG_MAIN
 
 #include "catch.hpp"
```

Comment:
Was this file intended to be (mostly) empty?

Reply:
For that one, I actually just looked to see what ENDFtk did with the analogous file. It did exactly this. :-) I guess the question is whether or not we need to "test" the main, GNDStk.hpp header file that everyone includes. Maybe we should just delete this??

Reply:
If there is something that can only be tested at the highest level, then it should go here. If there is nothing, then let's delete the file—and perhaps even the `test` directory.